src/core/toga/widgets/icon.py

```diff
@@ -39,10 +39,11 @@ def __init__(self, path, system=False):
     def filename(self):
         if self.system:
             toga_dir = os.path.dirname(os.path.dirname(__file__))
+            print(toga_dir)
             return os.path.join(toga_dir, 'resources', self.path)
         else:
-            from toga.app import App
-            return os.path.join(App.app_dir, self.path)
+            # from toga.app import App
+            return os.path.join("", self.path)
```

Comment:
Is this change intentional? It looks like something that might have been commented out while you were exploring some aspect of the API.

Reply:
Yes. It was intentional. I was getting an error with the "App.app_dir" because there was no instance of App, hence "app_dir" was not available.

Reply:
Right - so there is an underlying bug that needs to be fixed, not just papering over the cause.

Reply:
Sure. I am just starting out with open source, so i am still new to some of these thing. But will keep that in mind for later.